easybuild/easyblocks/v/vep.py

```diff
@@ -105,9 +112,13 @@ def make_module_req_guess(self):
         """Custom guesses for environment variables (PATH, ...) for VEP."""
         perl_majver = get_major_perl_version()
 
+        perl_libpath = [self.api_mods_subdir]
+        if 'Bio::EnsEMBL::XS' in [ext[0] for ext in self.cfg['exts_list']]:
+            perl_libpath += ['lib/perl%s/site_perl/%s' % (perl_majver, get_software_version('Perl'))]
```

Comment:
same here, please use `.extend` and `os.path.join`

Reply:
```python
perl_maj_ver = get_major_perl_version()
perl_ver = get_software_version('Perl')
perl_libpath.extend(os.path.join('lib', 'perl' + perl_maj_ver, 'site_perl', perl_ver)

Reply:
Fixed as well :+1: